lstchain/tests/test_lstchain.py

```diff
@@ -122,6 +122,36 @@ def test_apply_models():
     dl2.to_hdf(dl2_file, key=dl2_params_lstcam_key)
 
 
+@pytest.mark.run(after='test_apply_models')
+def test_sensitivity():
+    from lstchain.mc import sensitivity 
+    
+    nfiles_gammas = 1
+    nfiles_protons = 1
+
+    eb = 20  # Number of energy bins
+    gb = 11  # Number of gammaness bins
+    tb = 10  # Number of theta2 bins
+    obstime = 50 * 3600 * u.s
```

Comment:
`import astropy.units as u` at the top of the file

Reply:
I've been playing recently with the code and actually they should be dl1 files because the mc info is stored also in dl1, no need to read simtel files anymore. Now I'm thinking that probably those two dl1 files are unnecesary since mc info is also stored in dl2 files, am I right?

Reply:
I think you are absolutely right, the code actually tracks down to `read_simu_info_merged_hdf5` that takes hdf5 files as input.
So `sensitivity` and `find_best_cut_sensitivity` can be a lot simplified.

Reply:
added